setup.py

```diff
@@ -22,15 +22,10 @@
     'pymongo>=3.7.2,<4',
     'scikit-learn>=0.21',
     'tqdm<4.50.0,>=4.36.1',
-    'cloudpickle>=1.6,<2',
     'scipy>=1.0.1,<2',
-    'numpy<1.19.0,>=1.16.0',
-    'pandas>=1,<2',
```

Comment:
Is there a specific reason we're removing these instead of just unpinning them? It feels weird to me to entirely remove a first order dependency, even if we know that another dependency will install them and be more restrictive. I'm not sure what best practice is in this case.

Reply:
That makes sense. My logic is that `mlprimitives` installs all of these packages and Draco should have a sole dependence on `mlprimitives`. It doesn't hurt to have them there, I think it is just easier for me to see which package is installing what from all the dependencies.

But yeah, I don't know what the best practice is

Reply:
Yeah that makes sense. I think from a "purist" point of view, it makes sense to me to think of first order dependencies as black boxes and not assume sub-dependencies or implementation details. So since we import `numpy` and `pandas`, we shouldn't assume they're sub-dependencies of one of our dependencies. On the other hand, since we control `mlprimitives` and know that it will install them and be more restrictive than we are, it might be ok in this case. @dyuliu do you have any thoughts?

Reply:
I think your point of you makes sense. Will update!